frontend/assets/stylesheets/components/_packages.scss

```diff
@@ -169,8 +220,7 @@ $_package-promo__actions_height: 117px;
         }
         .package-promo__actions {
             position: static;
-            padding-top: rem($gs-gutter / 4);
-            padding-bottom: rem($gs-gutter / 4);
+            padding: rem($gs-gutter / 4);
```

Comment:
you can remove these `rem()` functions from any code you touch, we have the px-to-rem plugin now